packages/sitecore-jss/src/personalize/layout-personalizer.test.ts

```diff
@@ -165,5 +165,29 @@ describe('layout-personalizer', () => {
       );
       expect(personalizedComponentResult).to.equal(null);
     });
+
+    it('should return personalized component without experiences', () => {
```

Comment:
This test case and another one below can be verified in our existing unit tests
"should return array of personalized components"
"should return default component when variant is undefined"

Reply:
I agree on the first one, but am not sure about "should return default component when variant is undefined".
There we don't explicitly check if experiences are empty. it would also pass before the changes. So maybe we should keep the new test for default component?

Reply:
I mean that "should return default component when variant is undefined" should be extended to check experiences as a first test case

Reply:
Both tests can be merged with existing (above)